src/serac/physics/heat_transfer.hpp

```diff
@@ -871,7 +874,7 @@ class HeatTransfer<order, dim, Parameters<parameter_space...>, std::integer_sequ
    */
   FiniteElementDual& computeTimestepSensitivity(size_t parameter_field) override
   {
-    auto drdparam     = serac::get<DERIVATIVE>(d_residual_d_[parameter_field]());
+    auto drdparam     = serac::get<DERIVATIVE>(d_residual_d_[parameter_field](ode_time_point_));
```

Comment:
I just put some big TODOs next to this line, since we don't have tests to confirm if this part of the code is functioning or not. We'll look into it more closely when refactoring the time integration procedures.